windows/src/global/inc/keymanengine.h

```diff
@@ -77,6 +87,9 @@ typedef struct tagINTKEYBOARDINFO
   LPINTKEYBOARDOPTIONS KeyboardOptions;
   int        nProfiles;
   LPINTKEYBOARDPROFILE Profiles;
+  km_kbp_keyboard* coreKeyboard;
+  km_kbp_option_item* coreKeyboardOptions;
+  km_kbp_state* lpActiveKBState;
```

Comment:
To match other member name styles, can we go with something like:

```suggestion
  km_kbp_keyboard* CoreKeyboard;
  km_kbp_option_item* CoreKeyboardOptions;
  km_kbp_state* CoreKeyboardState;
```

or perhaps better:

```suggestion
  km_kbp_keyboard* lpCoreKeyboard;
  km_kbp_option_item* lpCoreKeyboardOptions;
  km_kbp_state* lpCoreKeyboardState;
```